docs/reading.md

```diff
@@ -18,7 +18,7 @@ adheres to as well.
 
 ## Schema inference
 
-The MarkLogic Spark connector will infer a Spark schema automatically based on the view that is used referenced in 
+The connector will infer a Spark schema automatically based on the view that is used referenced in 
```

Comment:
The wording here is weird. Is it a typo?

Reply:
Yup, was a lingering typo. Fixed.